tree/dataframe/inc/ROOT/RDF/RVariation.hxx

```diff
@@ -161,13 +161,10 @@ class R__CLING_PTRCHECK(off) RVariation final : public RVariationBase {
          throw std::runtime_error("The evaluation of the expression for variation \"" + variationName +
                                   "\" resulted in " + std::to_string(GetNVariations(results)) + " values, but " +
                                   std::to_string(fVariationNames.size()) + " were expected.");
+         (void)entry; // avoid unused parameter warnings (gcc 12.1)
```

Comment:
Why is this in the `if` clause? Consider moving this up to line 158, i.e. right after the potential use `Get<ColTypes>(entry)`

Reply:
absolutely, thanks!

Reply:
done